src/client/components/UserEditor.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import { User } from '../../shared/User';
+import { remult } from '../utils/api-facade';
+import { Box } from '@mui/material';
+import { ErrorInfo } from 'remult';
```

Comment:
TS < 4.5
```suggestion
import type { ErrorInfo } from 'remult';
```

TS >= 4.5
```suggestion
import { type ErrorInfo } from 'remult';
```